<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_txt"/>
    <addaction name="action_txt_2"/>
    <addaction name="action_bin"/>
    <addaction name="action_bin_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_txt">
   <property name="text">
    <string>Сохранить в txt</string>
   </property>
  </action>
  <action name="action_txt_2">
   <property name="text">
    <string>Загрузить из txt</string>
   </property>
  </action>
  <action name="action_bin">
   <property name="text">
    <string>Сохранить в bin</string>
   </property>
  </action>
  <action name="action_bin_2">
   <property name="text">
    <string>Загрузить из bin</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
